src/pages/ChatSelector/ChatSelector.jsx

```diff
@@ -0,0 +1,54 @@
+import { useNavigate } from 'react-router';
+import { usePublicChatRooms, useUser, useUserChatRooms } from '../../hooks';
+import { Button, ChatRoomTile, Text } from '../../components';
+
+import './ChatSelector.scss';
+import { routes } from '../../routes/Routes';
+
+export const ChatSelectorPage = () => {
+    const user = useUser();
+    const userChatRooms = useUserChatRooms(user.data.uid);
```

Comment:
you could use the desctuct

const {data: userChatRooms} = useUserChatRooms() etc.